client/js/widgets/label.js

```diff
@@ -27,6 +28,11 @@ export class Label extends Widget {
   applyDeltaToDOM(delta) {
     super.applyDeltaToDOM(delta);
     if(delta.text !== undefined || delta.twoRowBottomAlign !== undefined) {
+      if (this.get('spellCheck') === true) {
+        this.input.setAttribute('spellcheck', true);
+      } else {
+        this.input.setAttribute('spellcheck', false);
+      }
```

Comment:
This if/else block can be shortened to just:
```js
this.input.setAttribute('spellcheck', this.get('spellCheck') === true)
```

Reply:
> This if/else block can be shortened to just:
> 
> ```js
> this.input.setAttribute('spellcheck', this.get('spellCheck') === true)
> ```

chatGPT says this.input.setAttribute('spellcheck', this.get('spellCheck')) is enough

Reply:
That would set the spellcheck attribute to exactly what was entered as the spellCheck property which could be arbitrary strings, objects which would be coerced into the string `[object Object]` etc. You definitely want to force it to be a boolean and @96LawDawg also wanted it to only set spellcheck = true if the value specified is actually true, rather than arbitrary strings.

Reply:
I'm trying to avoid a situation where a label already has a spellCheck property and need for a file updater.  No matter what, this isn't game breaking, but just trying to avoid conflict.